client/stylesheets/_toCat.scss

```diff
@@ -5464,12 +5479,16 @@ On click the images will be displayed at normal size to complete the effect
     margin-left: 0;
 }
 
+.profileUserAmount ul li span {
```

Comment:
Merge rule `.profileUserAmount ul li span` with rule on line 5468<br>Selector should have depth of applicability no greater than 2, but was 4<br>Selector `profileUserAmount` should be written in lowercase with hyphens